tslint.json

```diff
@@ -49,11 +49,7 @@
     ],
     "max-classes-per-file": false,
     "max-line-length": [true, 140],
-    "member-ordering": [true,
-      "public-before-private",
-      "static-before-instance",
-      "variables-before-functions"
-    ],
+    "member-ordering": [true, { "order": "fields-first" }],
```

Comment:
is the old options syntax still supported?

Reply:
oh right, you asked. we cannot ship a breaking change. please preserve the old syntax, or we'll have to talk about 6.0